### ISSUE TITLE
Load and Unload never called for ElementRenderer for UI?

### ISSUE THREAD

**Issue by Uniphix** [SiliconStudio/paradox#145]
Is this a bug?

Load and Unload never called for ElementRenderer for UI?  I noticed when doing any references to it 

https://github.com/SiliconStudio/paradox/blob/master/sources/engine/SiliconStudio.Paradox.UI/Renderers/ElementRenderer.cs

**Comment by xen2** [comment]
Seems like a bug, good catch, thanks!

**Comment by Uniphix** [comment]
Your very welcome, I'll keep finding those.  Reason why I caught it was because I am using the same similar system for our "Cursor3D" system

**Comment by xenux** [comment]
Yes, currently the Load and Unload of ElementRenderer are not correctly called each time a ElementRenderer is added and removed from the pipeline. It was actually in my list of task. It requires some modifications in the RendererManager. I will try to do it soon. Thank you for the report anyway.
